Added Terminology and Workflow files for Git (#10)
helicalco/docs: content/learn/git/terminology.md

---
title: Terminology
pcx-content-type: tutorial
weight: 1
meta:
  title: Terminology
---

# Terminology

### Repository

A repository can be thought of as a directory that stores all versions of files, folders and content regarding a project.

### Clone/Cloning

When you clone a repository you are creating a copy of the repository on your computer.

### Origin (Remote)

> An alias for the remote repository that is used to track changes - the repository you will typically clone from. It is the repository that is used to push and pull changes. It is a shared repository that all users can access and exchange their changes. Stored on a hosting service like github or gitlab.

Reviewer: ```suggestion
An alias for the remote repository that is used to track changes - the repository you will typically clone from, and then push and pull changes. It is a shared repository that all users can access and exchange their changes. Stored on a hosting service like Github.
```
